include/vinecopulib/vinecop/implementation/fit_controls.ipp

```diff
@@ -24,6 +24,7 @@ inline FitControlsVinecop::FitControlsVinecop()
   select_families_ = true;
   show_trace_ = false;
   mst_algorithm_ = "prim";
+  set_seeds(std::vector<int>());
```

Comment:
I'm a bit unhappy about the (in 99.99999% of cases unnecessary) calls to `set_seeds()` here and the other constructors. With default values, this queries random numbers from the default random device (on some machines by asking the operating system for some, IIRC). Can we make this conditional on the algorithm being random?

Reply:
I was thinking about this and it's not really a computational bottleneck, and having ifs made things heavier than needed. I would also need to reseed or ensure things are properly seeded if the set_mst_algorithm method is used, which felt unnecessary.